src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -40,6 +47,49 @@ const useStyles = makeStyles((theme) => ({
 
 export default function EnvironmentEditor() {
     const classes = useStyles();
+    const [scenes, setScenes] = useState([
+        {
+            id: "1",
+            name: "Scene 1",
+        },
+        {
+            id: "2",
+            name: "Scene 2",
+        },
+        {
+            id: "3",
+            name: "Scene 3",
+        },
+        {
+            id: "4",
+            name: "Scene 4",
+        },
+        {
+            id: "5",
+            name: "Scene 5",
+        },
+    ]);
+
+    const reorder = ({ scenes, startIndex, endIndex }) => {
+        const result = Array.from(scenes);
+        const [removed] = result.splice(startIndex, 1);
+        result.splice(endIndex, 0, removed);
+
+        return result;
+    };
```

Comment:
Should this function be defined outside or inside the component? Not sure what the best practice is here.

Reply:
Not sure either. We can keep it here for now and refactor if it will need to be used elsewhere.